auth-web/src/components/auth/account-settings/statement/StatementsSettings.vue

```diff
@@ -286,7 +286,11 @@ export default {
         await orgStore.syncActiveOrgMembers()
         await orgStore.fetchStatementSettings()
         await orgStore.getStatementRecipients()
-        state.frequencySelected = state.statementSettings?.currentFrequency?.frequency || state.statementSettings?.frequencies[0].frequency
+        if (isEFT) {
+          state.frequencySelected = state.statementSettings?.frequencies[2].frequency
```

Comment:
what does 2 represent?

Why not use find and get the correct frequency instead?